src/main/java/com/softserve/demo/service/impl/OfferServiceImpl.java

```diff
@@ -51,4 +44,14 @@ public void deleteOffer(Integer id) {
     public List<Offer> getOffersByCustomerId(Integer id) {
         return offerRepository.findAllByCustomerId(id);
     }
+
+    private Location saveLocation(final Location location) {
```

Comment:
This method is duplicated in OrderServiceImpl. Move it to LocationService and reuse in both places